config/routes.rb

```diff
@@ -22,6 +23,8 @@
   namespace :api do
     namespace :v1 do
       get 'links/create'
+      get 'links/show' => 'api/links'
```

Comment:
Is this expecting a query parameter to be passed to find the link? This can be handled on the URI itself by making the routes expect an `:id` in it.  Making it `/api/v1/links/:id` would make it a TRUE RESTful route :smile:

Same for the `api/v1/users/:id` route.